Reports/ProgressReport-Feb24-Aug24.md

```diff
@@ -0,0 +1,332 @@
+
+This section contains project plans and priorities , currently focusing on work to be done over the next month it should eventually encompass longer term plans.
+
+### Monthly priorities: April
+
+The list of priority stories can eb found here https://github.com/orgs/alan-turing-institute/projects/111/views/11
+
+#### Communication and outreach [35](https://github.com/alan-turing-institute/data-safe-haven-team/issues/35) 
+
+Follow up on conversations from AI UK to keep momentum going, as resources are limited this month the work here should be to establish connections buyt any meetings or further work left for furhter ahead.
+
+#### Project Strategy and Ways of work [43](https://github.com/alan-turing-institute/data-safe-haven-team/issues/43)
+
+@davsarper keen to prioritise as possible to:
+
+- Create due reports
+- Revisit reports and monthlies format
+- Propose and discuss PIs WoW
+
+#### Manage codebase releases adn testing [50](https://github.com/alan-turing-institute/data-safe-haven-team/issues/50)
+
+This story was a March priority and the release completed, with less time dedicated there will be follow up actions. Including any fixes and also required communications about the release.
+
+#### Identify and implement core IAC changes [28](https://github.com/alan-turing-institute/data-safe-haven-team/issues/28)
+
+April work will focus on identifying and planning work within this story so it can be a top priority in May. 
+
+#### UK TRE Community [52](https://github.com/alan-turing-institute/data-safe-haven-team/issues/52)
+
+Having finalised the funding phase there is three important things to do:
+- Wrap up outputs, and celebrate WGs event
+- Transition and knowledge transfer: ensure the community can continue delivering work
+
+#### Manage  grant opportunities and pipeline [51](https://github.com/alan-turing-institute/data-safe-haven-team/issues/51)
+
+It is a priority to at least think and plan, this story can be directly linked to PI work and how we better coordinate. So not a high resources story but still very important one
+
+#### DSPT submission [37](https://github.com/alan-turing-institute/data-safe-haven-team/issues/37)
+
+Finalise submission, it should not require intensive effort but it will include revision by project PIs and possibly Data Protection (as Manager changed between submissions).
+
+New non-mandatory requirements that can be easily completed will be (already identified).
+
+#### TRESA costs and cost recovery [36](https://github.com/alan-turing-institute/data-safe-haven-team/issues/36)
+
+While capacity is stretched with the new year we have to clarify recharges process and prepare for the announcement in May's catch up.
+
+#### Not doing or prioritising
+
+This month we are not leaving much out (please add anything you believe we should keep in mind), with the new FY we are doing a bit everywhere in a month characterised by planning ahead.
+
+### Next up: upcoming priorities and work the will be necessary
```

Comment:
Where are we planning to report planning for @harisood moving on? that should be a fair chunk of work this coming month no?

Reply:
I agree this can be a good place